config.py

```diff
@@ -0,0 +1,111 @@
+import os
+import types
+from functools import reduce
+
+_GLOBAL_CONFIG = dict(
+    # If running on Tools labs, keep database dumps in this directory...
+    archive_dir = os.path.join(os.path.expanduser('~'), 'archives'),
+
+    # ...and delete dumps that are older than this many days
+    archive_duration_days = 90,
+
+    # Where to put various logs
+    log_dir = os.path.join(os.path.expanduser('~'), 'logs'),
+
+    flagged_off = [],
+
+    profile = True,
+
+    stats_max_age_days = 90,
+
+    user_agent = 'citationdetective',
+
+)
+
+_CITATION_NEEDED_CONFIG = dict(
```

Comment:
The first three here only apply to English, right? So they could be part of the English-specific config, and `max_seq_length` could be global.

Reply:
Yes, the first three apply to English. I've checked `max_seq_length` also differs from language to language model, so I'll put them to English-specific config.